lib/rubocop/cop/rspec/factory_bot/consistent_parentheses_style.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      module FactoryBot
+        # Use a consistent style for parentheses in factory bot calls
+        #
+        # @example
+        #
+        #   # bad
+        #
+        #   create :user
+        #   build(:user)
+        #   create(:login)
+        #   create :login
+        #
+        #   # good
+        #   create :user
+        #   build :user
+        #   create :login
+        #   create :login
+        #
+        #   # good
+        #   create(:user)
+        #   create(:user)
+        #   create(:login)
+        #   build(:login)
+        #
+        class ConsistentParenthesesStyle < Base
+          extend AutoCorrector
+          include ConfigurableEnforcedStyle
+
+          MSG_ENFORCE_PARENS = 'Prefer method call with parentheses'
+          MSG_OMIT_PARENS = 'Prefer method call without parentheses'
+
+          def_node_matcher :factory_call, <<-PATTERN
+            {
+              (send
+                ${(const nil? {:FactoryGirl :FactoryBot}) nil?} :create (sym $_)
```

Comment:
`(sym $_)` limits this to literal symbols:
```
create(:user)
```
however, it will miss other cases:
```
model = :user
create(model)
```
or
```ruby
create('user')
```